owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListFragment.kt

```diff
@@ -1357,6 +1370,52 @@ class MainFileListFragment : Fragment(),
         }
     }
 
+    private fun showRemoveCustomDialog(file: OCFile, context: Context) {
+        val removeDialog = Dialog(context)
+        removeDialog.apply {
+            setContentView(R.layout.remove_files_dialog)
+            setCancelable(false)
+            show()
+        }
+
+        val imageView = removeDialog.findViewById<ImageView>(R.id.dialog_remove_thumbnail)
+        val content = removeDialog.findViewById<TextView>(R.id.dialog_remove_information)
```

Comment:
Let's set more accurate variable names, like `thumbnailImageView` and `dialogMessage` (examples), so that when we find the variable in the code we know what it refers to 👍. `content` could lead to a misunderstanding thinking it refers to a container layout, for example